lib/mayaUsd/ufe/UsdHierarchy.cpp

```diff
@@ -338,6 +401,10 @@ Ufe::SceneItem::Ptr UsdHierarchy::defaultParent() const
 
 Ufe::UndoableCommand::Ptr UsdHierarchy::reorderCmd(const Ufe::SceneItemList& orderedList) const
 {
+    // Changing the hierarchy of inactive items is not allowed.
+    if (!fItem->prim().IsActive())
+        return nullptr;
+
```

Comment:
You're saving a bit of work by putting the test before the loop, but it would still be safer to put the test in UsdUndoReorderCommand::create().